### PR TITLE
Click tracking

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @zeantsoi @bsimpson63 

related: https://github.com/reddit/reddit-public/pull/4130

### CODE REVIEW

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -202,6 +209,17 @@ def update_creative(link, az_advertiser):
     return az_creative
 
 
+def update_trackers(campaign, az_cfmap):
+    if (not getattr(campaign, "adserver_click_url", False) or
+        not getattr(campaign, "adserver_impression_url", False)):
+
+        pixels = az_cfmap._get_pixel()
+
+        campaign.adserver_click_url = pixels['StaticClickUrl']
```
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: Do we want/need to set defaults for `adserver_click_url` and `adserver_impression_url` on `PromoCampaign`?

Comment (dwick) [inline reddit_adzerk/adzerkpromote.py]: we don't need to.  i'm not super clear on when we add defaults, this isn't tied to the plugin so i suppose we could... @bsimpson63 thoughts?

Comment (bsimpson63) [inline reddit_adzerk/adzerkpromote.py]: I think we've been not adding defaults for attributes that only exist in a plugin.

**Comment by bsimpson63** [comment]
:fish:

**Comment by dwick** [comment]
218c48ef405c57e5cc43c3ac342585144854a7ef 95c13f43263c5b2d21f7223685a3e42c02e3d214 79bc7cb34e32e7a595953e187d303e6e9253d2ee

**Pull request closed without merging**